tests/utils.py

```diff
@@ -178,3 +178,15 @@ def wrapper():
         used.append(item)
         return item
     return wrapper
+
+
+def assert_mutual_in(list_one, list_two):
```

Comment:
Cool, thanks. I looked through, but had a hard time finding something that was quite right.